### DISCUSSION TITLE
smart-pointers/box

### DISCUSSION THREAD

**Discussion by giscus** [sunface/rust-by-practice#247] [Book Comments]
# smart-pointers/box

Learning Rust By Practice, narrowing the gap between beginner and skilled-dev with challenging examples, exercises and projects.

https://zh.practice.rs/smart-pointers/box.html

**Comment by 9a-aaaaaaaa** [comment]
i am a looser !

**Comment by Tlntin** [comment]
催更

**Comment by blurskye** [comment]
i am really enjoying this, please use ai or otherise, finish this series, i see it was up for like 2 year and it wasnt done
: \

**Comment by XueHua-s** [comment]
为啥没练习
